core/src/main/java/contacts/core/data/DataQuery.kt

```diff
@@ -18,149 +18,150 @@ import contacts.core.util.unsafeLazy
  * Provides new query instances for specific types of Profile OR non-Profile (depending on instance)
  * data.
  */
-interface DataQuery {
+interface DataQueryFactory {
```

Comment:
Added "Factory" suffix as it more accurately describes what this is doing, which is creating instances of the actual API that does the query; `DataQuery`.